packages/core/test/validate_test.js

```diff
@@ -374,6 +385,11 @@ describe("Validation", () => {
       expect(errors).not.to.be.empty;
       expect(errors[0].message).to.equal(newErrorMessage);
     });
+
+    it("should allow object error messages", () => {
+      expect(errors).not.to.be.empty;
```

Comment:
Would this test fail if the proptype was set to a string? Or would it merely emit a warning to the console?

Reply:
None of those, after reviewing it , this test is not really enforcing the propTypes validation. I will update the test cases. It should be focus on SchemaField or any other field, as it is there where the propTypes validation occurs.